gradebookng/tool/src/java/org/sakaiproject/gradebookng/GradebookNgApplication.properties

```diff
@@ -385,6 +385,7 @@ grade.notifications.privileges = Insufficient privileges to edit this Gradebook
 grade.notifications.revert = Revert to Previous Score
 grade.notifications.saved = All changes saved.
 grade.notifications.error = Errors were detected. See cell notifications below.
+grade.notifications.excused = Item has been excused from grade calculations.
```

Comment:
Not sure if "item" (could be understood to mean the entire column) is the best wording here. "Grade" might be more clear. There are ways for the entire column to be excluded.